### ISSUE TITLE
Documentation or Mechanism for Updating a Running Service 

### ISSUE THREAD

**Issue by christopherhein** [metaparticle-io/package#12]
Trying to figure out the strategy for updating a service beyond, deleting or manually using `kubectl`. 

If the `docker` tag updates when I use something like `npm start` shouldn't the deployment also get updated?

**Comment by brendandburns** [comment]
Yes, this is a very interesting question. I'd love to see a proposal for how we might do this in the form of quick mark-down document...
